### DISCUSSION TITLE
Error using mpiexec

### DISCUSSION THREAD

**Discussion by Khaksarkn** [idaholab/moose#25243] [Q&A General] [answered]
Hi, 

I wrote a coupled THM code using MOOSE and it works fine on a single core, but when I want to run it in multicore using "mpiexec", I get this error:

mpiexec -n 4 ./frozen_soil_thm -i ./problems/THMTests4_coupled_verification/input.i 

[proxy:0:0@Kians-MacBook-Air.local] HYDU_create_process (utils/launch/launch.c:73): execvp error on file ./frozen_soil_thm (No such file or directory)
[proxy:0:0@Kians-MacBook-Air.local] HYDU_create_process (utils/launch/launch.c:73): execvp error on file ./frozen_soil_thm (No such file or directory)
[proxy:0:0@Kians-MacBook-Air.local] HYDU_create_process (utils/launch/launch.c:73): execvp error on file ./frozen_soil_thm (No such file or directory)
[proxy:0:0@Kians-MacBook-Air.local] HYDU_create_process (utils/launch/launch.c:73): execvp error on file ./frozen_soil_thm (No such file or directory)

This is a THM code and it wasn't an issue before adding the mechanical codes (when it was just a TH code) but now I'm getting this. 

I would appreciate it if you could let me know what the problem might be. 
Thanks.

**Comment by loganharbour** [comment, marked as answer]
You're getting this error because `frozen_soil_thm` does not exist. It shouldn't work in serial without mpi either.

Make sure you're in the right directory and make sure your application is compiled.

**Reply by Khaksarkn** [reply]
Thank you. Yes, it was a silly mistake!
